OpenRA.Game/Traits/Selectable.cs

```diff
@@ -33,6 +38,10 @@ public Selectable(Actor self, SelectableInfo info)
 		{
 			this.self = self;
 			Info = info;
+			if (string.IsNullOrEmpty(info.Class))
```

Comment:
This could become ternary, but that doesn't change the actual logic so it isn't necessary.